core/src/window_service.rs

```diff
@@ -137,17 +142,58 @@ impl WindowServiceMetrics {
     }
 }
 
+fn should_send_index_and_erasure_conflicts(
+    shred_slot: Slot,
+    feature_set: &FeatureSet,
+    epoch_schedule: &EpochSchedule,
+) -> bool {
+    match feature_set.activated_slot(&feature_set::index_erasure_conflict_duplicate_proofs::id()) {
+        None => false,
+        Some(feature_slot) => {
+            let feature_epoch = epoch_schedule.get_epoch(feature_slot);
+            let shred_epoch = epoch_schedule.get_epoch(shred_slot);
+            // Has a 1 epoch delay, as we don't have enough information
+            // on the epoch boundary of the feature activation
+            feature_epoch < shred_epoch
+        }
+    }
+}
+
 fn run_check_duplicate(
     cluster_info: &ClusterInfo,
     blockstore: &Blockstore,
     shred_receiver: &Receiver<PossibleDuplicateShred>,
     duplicate_slots_sender: &DuplicateSlotSender,
+    bank_forks: &Arc<RwLock<BankForks>>,
```

Comment:
`&Arc` is redundant. Either just `&` or if you really need `Arc` then just `Arc`.